client/src/interface/User.ts

```diff
@@ -2,6 +2,9 @@ export interface User {
   id: string;
   email: string;
   profilePicture: string;
+  board: {
```

Comment:
Don't forget to remove this before merging after refactoring the code to use the existing Board, Column, and Card interfaces.